### ISSUE TITLE
Maybe try posting this to PowerToys GitHub to get Microsoft's attention to correct SDR gamma

### ISSUE THREAD

**Issue by Jason-GitH** [dylanraga/win11hdr-srgb-to-gamma2.2-icm#9]
Thanks for investigating this!
I think it would be worthwhile to submit a request to [Windows PowerToys GitHub](https://github.com/microsoft/PowerToys) to have someone at Microsoft take notice. I would hope that user @mdrejhon who works with monitor manufacturers and may be able to provide some insight.

**Comment by dylanraga** [comment]
That would be an excellent avenue for this type of feature, I'll look into it. Unfortunately, sRGB is Microsoft's baby, so I have doubts they'll budge on it.

**Comment by Jason-GitH** [comment]
> That would be an excellent avenue for this type of feature, I'll look into it. Unfortunately, sRGB is Microsoft's baby, so I have doubts they'll budge on it.

I submitted some feedback to Microsoft via the Feedback hub and they deleted all of them and instead assigned an upvote to these already submitted Feedbacks.  A Microsoft engineer acknowledged the first one 10 months and said "We're looking into it." It would be great if more people upvoted these since these seem to be the ones Microsoft is focusing on.

[Feedback 1](https://aka.ms/AAmsm0h)
[Feedback 2](https://aka.ms/AAmt112)

**Comment by morphinapg** [comment]
> > That would be an excellent avenue for this type of feature, I'll look into it. Unfortunately, sRGB is Microsoft's baby, so I have doubts they'll budge on it.
> 
> I submitted some feedback to Microsoft via the Feedback hub and they deleted all of them and instead assigned an upvote to some already submitted Feedbacks listed below. A Microsoft engineer acknowledged the first one 10 months and said "We're looking into it." It would be great if more people upvoted these since these seem to be the ones Microsoft is focusing on.
> 
> [Feedback 1](https://aka.ms/AAmsm0h) [Feedback 2](https://aka.ms/AAmt112)

I just commented a suggestion that they could implement that would make workarounds like this unnecessary. A gamma setting, with presets of 2.2, 2.4, and sRGB, as well as a custom slider with possible calibration pattern like those you see in a game.

And also, allowing the use of "SDR Only" and "HDR Only" ICC profiles, so that each mode could be precisely calibrated separately.

**Comment by Jason-GitH** [comment]
> > > That would be an excellent avenue for this type of feature, I'll look into it. Unfortunately, sRGB is Microsoft's baby, so I have doubts they'll budge on it.
> > 
> > 
> > I submitted some feedback to Microsoft via the Feedback hub and they deleted all of them and instead assigned an upvote to some already submitted Feedbacks listed below. A Microsoft engineer acknowledged the first one 10 months and said "We're looking into it." It would be great if more people upvoted these since these seem to be the ones Microsoft is focusing on.
> > [Feedback 1](https://aka.ms/AAmsm0h) [Feedback 2](https://aka.ms/AAmt112)
> 
> I just commented a suggestion that they could implement that would make workarounds like this unnecessary. A gamma setting, with presets of 2.2, 2.4, and sRGB, as well as a custom slider with possible calibration pattern like those you see in a game.
> 
> And also, allowing the use of "SDR Only" and "HDR Only" ICC profiles, so that each mode could be precisely calibrated separately.

Yes, I realize now that is very important. I tested out your profiles and there is a loss of about 30-40 nits of PQ values in near black detail in native HDR10 content.

**Comment by Jason-GitH** [comment]
> That would be an excellent avenue for this type of feature, I'll look into it. Unfortunately, sRGB is Microsoft's baby, so I have doubts they'll budge on it.

I posted this request here: https://github.com/microsoft/PowerToys/issues/29949
I am not sure I can handle their responses, as I am not knowledgeable about color science as you are and don't know how to program. They seem only interested in projects that can utilize existing Windows APIs.
